### PR TITLE
javascript-light-bulb - Michael Suh

### PR DESCRIPTION
**Pull request by michael-suh**
js lightbulb SR

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by michael-suh** [comment]
js lightbulb SR

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 var $centerButton = document.querySelector('.center-button');
 var $backgroundColor = document.querySelector('.background-color');
 
 function clickButton(event) {
-  numberOfClicks++;
-  if (numberOfClicks % 2 === 0) {
-    $centerButton.className = 'center-button dark';
-    $backgroundColor.className = 'background-color dark';
-  } else {
+  if ($backgroundColor.className === 'background-color dark') {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:5]: This is making your JavaScript dependent on the state of the DOM to decide what action to take, and that is bad. You want to be able to know whether to turn the light on or off based on the value of a variable. I encourage you to stick with the boolean path I described in my last review.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 1 above)
Comment (michael-suh) [reply javascript-light-bulb/main.js:5]: I actually don't really understand what I have to do. To be honest, I thought this was the boolean path :/ Do you mean I have to set a new variable? for ex, var lightbulb = on || off?

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**